api/templates/design.go

```diff
@@ -181,10 +181,11 @@ var _ = Service("templates", func() {
 		Meta("openapi:summary", "List templates in the repository")
 		Description("List all templates  defined in the repository.")
 		Result(Templates)
-		Payload(RepositoryRequest)
+		Payload(TemplatesRequest)
 		HTTP(func() {
 			GET("/repositories/{repository}/templates")
 			Meta("openapi:tag:template")
+			Param("filterBy")
```

Comment:
```suggestion
			Param("filter")
```
I would expect that I use `/repositories/keboola/templates?filter=keboola.data-apps`. the `filterBy=keboola.data-apps` seems weird but it is just my weird taste :smile:

Reply:
[770aa80](https://github.com/keboola/keboola-as-code/pull/2144/commits/770aa8021ee46f9f69bc9fe881453872b36d593c)